CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 
 ## [Unreleased]
 
+## [v4.2.0] - 2021-06-24
+
+### Added
+
+- cmd/definitions: Implement GSP-97 Add Restrictions In Storage Metadata (#601)
+- cmd/definitions: Implement GSP-111 Add System Metadata in Storage Metadata (#605)
+
+### Changed
+
+- pkg/endpoint: Deprecated by github.com/beyondstorage/go-endpoint (#595)
+- cmd/definitions: Trim Service Pair Prefix (#598)
+- services: Implement GSP-90 Re-support Initialization Via Connection String (#589)
+- cmd/definitions: Implement GSP-109 Redesign Features (#606)
+- cmd/definitions: Implement GSP-117 Rename Service to System as the Opposite to Global (#608)
+
```

Comment:
Let's add docs for deprecated APIs.

Reply:
PTAL, I'm not sure whether it's necessary to list all the deprecated fields.

Reply:
The current style is OK to me.